create-liquid-app

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+# ensure that .log directory exists
+readonly log_dir="./.log"
+mkdir -p $log_dir
+
+readonly nigiri_config_path=$(realpath ./.nigiri)
```

Comment:
Are you planning to add this directory to the repository? Because if not then I don't see a reason why we couldn't stick with the default.

Reply:
You're right. Originally I anticipated having to modify some config files, but I ended up not needing to, so we can simplify this.